client/src/components/header/index.js

```diff
@@ -0,0 +1,108 @@
+          fill="#0DCCDE"
+        />
+      </svg>
+    }
+    servesTitle="Teeth Whitening"
+    servesDescription="Lorem, ipsum dolor sit amet consectetur adipisicing elit. Itaque necessitatibus quidem nesciunt dolor"
+  />,
+  <ServesCard
+    key="4"
+    coverIcon={
+      <svg
+        width="32"
+        height="45"
+        viewBox="0 0 32 32"
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+      >
+        <path
+          d="M30.2418 5.35544C30.2223 5.32028 30.195 5.28512 30.1637 5.25778C29.9684 5.06247 29.652 5.06247 29.4567 5.25778L22.7262 11.9882L20.0739 9.33591L26.8083 2.60153C26.8395 2.57028 26.8629 2.53903 26.8864 2.50387C27.027 2.26559 26.9489 1.9609 26.7106 1.82028C22.8747 -0.445345 17.8512 0.0741858 14.5504 3.37106C11.9333 5.98825 11.0661 9.69919 11.9528 13.0351L0.507474 24.4804C0.390287 24.5976 0.398099 24.789 0.519193 24.9101L7.1481 31.539C7.26919 31.6601 7.4645 31.6679 7.57779 31.5507L19.0192 20.1093C22.359 21 26.07 20.1367 28.6911 17.5156C31.9879 14.2148 32.5075 9.19137 30.2418 5.35544ZM26.7028 15.5234C24.4333 17.7929 21.027 18.2304 18.32 16.8359L17.9762 17.1797L17.9723 17.1757L7.26919 27.8828L4.17544 24.789L13.1637 15.8007C13.1637 15.8007 13.1637 15.8047 13.1676 15.8047L15.2301 13.7422C13.8356 11.0351 14.2731 7.62887 16.5426 5.35934C17.2746 4.62672 18.1561 4.06073 19.1269 3.70006C20.0977 3.33938 21.1349 3.19252 22.1676 3.2695L16.9762 8.457C16.7417 8.69174 16.61 9.00997 16.61 9.34176C16.61 9.67356 16.7417 9.99178 16.9762 10.2265L21.8395 15.0898C22.0742 15.3243 22.3925 15.456 22.7243 15.456C23.0561 15.456 23.3743 15.3243 23.609 15.0898L28.7965 9.90231C28.9411 11.9257 28.234 13.9961 26.7028 15.5234Z"
+          fill="#0DCCDE"
+        />
+      </svg>
+    }
+    servesTitle="Periodontics"
+    servesDescription="Lorem, ipsum dolor sit amet consectetur adipisicing elit. Itaque necessitatibus quidem nesciunt dolor"
+  />,
+];
+
+const Header = () => (
+  <div className="serves-display">
+    <div className="container">
+      <img src={theme} alt="dental cover pic" className="img" />
```

Comment:
There's a common component we created called Image you should use

Reply:
done